network/load-balancer/concepts.mdx

```diff
@@ -135,6 +135,11 @@ Proxy protocol is an internet protocol used to transfer connection information f
 
 ## Sticky session
 
-A sticky session enables the Load Balancer to bind a user’s session to a specific Instance. This ensures that all subsequent sessions from the user are sent to the same Instance, while there is at least one active session.
+A sticky session enables the Load Balancer to bind a user’s session to a specific Instance. This ensures that all subsequent sessions from the user are sent to the same Instance, while there is at least one active session. Sticky sessions can be **cookie based** or **table based**. 
```

Comment:
I would recommend using the term "server in the backend pool" rather than "Instance", as the console UI shows. A backend is a pool of servers that the load balancer can forward requests to. Using Instance could cause confusion as it could relate to the load balancer instance, or any Instance from Scaleway Elements.